middleware/src/middleware.go

```diff
@@ -1075,20 +1092,21 @@ func (middleware *Middleware) GetBaseInfo() rpcmessages.GetBaseInfoResponse {
 		ErrorResponse: &rpcmessages.ErrorResponse{
 			Success: true,
 		},
-		Status:              "-PLACEHOLDER-", // FIXME: This is a placeholder.
-		Hostname:            hostname,
-		MiddlewareLocalIP:   middlewareIP,
-		MiddlewareLocalPort: middlewarePort,
-		MiddlewareTorOnion:  middlewareTorOnion,
-		MiddlewareTorPort:   "-PLACEHOLDER-", // FIXME: This is a placeholder.
-		IsTorEnabled:        isTorEnabled,
-		IsBitcoindListening: isBitcoindListening,
-		FreeDiskspace:       freeDiskspace,
-		TotalDiskspace:      totalDiskspace,
-		BaseVersion:         baseVersion,
-		BitcoindVersion:     bitcoindVersion,
-		LightningdVersion:   lightningdVersion,
-		ElectrsVersion:      electrsVersion,
+		Status:                    "-PLACEHOLDER-", // FIXME: This is a placeholder.
+		Hostname:                  hostname,
+		MiddlewareLocalIP:         middlewareIP,
+		MiddlewarePort:            middlewarePort,
+		MiddlewareTorOnion:        middlewareTorOnion,
+		IsTorEnabled:              isTorEnabled,
+		IsBitcoindListening:       isBitcoindListening,
+		LightningActiveChannels:   lightningActiveChannels,
+		IsSSHPasswordLoginEnabled: isSSHPasswordLoginEnabled,
```

Comment:
Seems like `LightningActiveChannels` and `` are switched compared to the log output:

```
17:22:05 carol bbbmiddleware[2073]:  -PLACEHOLDER- carol 192.168.1.161 8845 5xla4g3brthhdyyg5ppxsvija3dysblv5tamealudpcu2lryj6k62oid.onion true true true 3 156599656448 502996557824 0.0.5 0.18.1 0.7.3 0.7.0}:
```

Reply:
Good eye, but struct fields are ordered. Values can however be assigned to these fields in a different order.

Could clean up the ordering in the assignment, but otherwise everything works as expected.

Reply:
Not an issue in this case. Just wanted to make sure that the `3` is really the Lightning number.